src/main/java/com/dre/brewery/configuration/ConfigManager.java

```diff
@@ -0,0 +1,254 @@
+                @Override public Class<? extends Annotation> annotationType() { return OkaeriConfigFileOptions.class; }
+                @Override public Class<? extends Configurer> configurer() { return BreweryXConfigurer.class; }
+                @Override public boolean useLangFileName() { return false; }
+                @Override public boolean update() { return false; }
+                @Override public String value() { return configClass.getSimpleName().toLowerCase() + ".yml"; }
+            };
+        }
+        return options;
+    }
+
+
+    // Not really what I want to do, but I have to move these from BConfig right now
+
+    public static void loadRecipes() {
+        // loading recipes
+        List<BRecipe> configRecipes = BRecipe.getConfigRecipes();
+        for (var recipeEntry : getConfig(RecipesFile.class).getRecipes().entrySet()) {
+            BRecipe recipe = BRecipe.fromConfig(recipeEntry.getKey(), recipeEntry.getValue());
+            if (recipe != null && recipe.isValid()) {
+                configRecipes.add(recipe);
+            } else {
+                BreweryPlugin.getInstance().errorLog("Loading the Recipe with id: '" + recipeEntry.getKey() + "' failed!");
+            }
+
+            BRecipe.setNumConfigRecipes(configRecipes.size());
+        }
+    }
+
+
+    public static void loadCauldronIngredients() {
+        // Loading Cauldron Recipes
+
+        List<BCauldronRecipe> configRecipes = BCauldronRecipe.getConfigRecipes();
+        for (var cauldronEntry : getConfig(CauldronFile.class).getCauldronIngredients().entrySet()) {
+            BCauldronRecipe recipe = BCauldronRecipe.fromConfig(cauldronEntry.getKey(), cauldronEntry.getValue());
+            if (recipe != null) {
+                configRecipes.add(recipe);
+            } else {
+                BreweryPlugin.getInstance().errorLog("Loading the Cauldron-Recipe with id: '" + cauldronEntry.getKey() + "' failed!");
+            }
+        }
+        BCauldronRecipe.setNumConfigRecipes(configRecipes.size());
+
+        // Recalculating Cauldron-Accepted Items for non-config recipes
+        for (BRecipe recipe : BRecipe.getAddedRecipes()) {
+            recipe.updateAcceptedLists();
+        }
+        for (BCauldronRecipe recipe : BCauldronRecipe.getAddedRecipes()) {
+            recipe.updateAcceptedLists();
+        }
+    }
+
+    public static void loadDistortWords() {
+        // Loading Words
+        Config config = getConfig(Config.class);
+
+        if (config.isEnableChatDistortion()) {
+            for (ConfigDistortWord distortWord : config.getWords()) {
+                new DistortChat(distortWord);
+            }
+            for (String bypass : config.getDistortBypass()) {
+                DistortChat.getIgnoreText().add(bypass.split(","));
+            }
+            DistortChat.getCommands().addAll(config.getDistortCommands());
+        }
+    }
+
+    public static void registerDefaultPluginItems() {
```

Comment:
Maybe we should make a `BreweryItem`ish type with a corresponding okaeri transformer?

Reply:
Don't understand this, sorry

Reply:
What I mean is that we could create an okaeri transformer that automatically transforms these strings into `Ingredient` or some other class

Reply:
If I'm understanding correctly, you want to create Transformers for things like the `BRecipe, BCauldronRecipe, and RecipeItem` class? I think I'd probably need to see an example of this to fully understand what you mean though